examples/jsm/nodes/pmrem/PMREMNode.js

```diff
@@ -162,7 +162,8 @@ class PMREMNode extends TempNode {
 
 		const texture = this.value;
 
-		if ( builder.renderer.coordinateSystem === WebGLCoordinateSystem && texture.isPMREMTexture !== true && texture.isRenderTargetTexture === true ) {
+		//if ( builder.renderer.coordinateSystem === WebGLCoordinateSystem && texture.isPMREMTexture !== true && texture.isRenderTargetTexture === true ) {
+		if ( texture.isRenderTargetTexture === true ) {
```

Comment:
Please avoid addressing multiple issues with a single PR.

We can leave it for now but in the future try fix different issues in different PRs please. That makes them easier to review and merge.

Reply:
I assumed that it had to be one PR because the changes in both files (PMREMGenerator and PMREMNode) belong together for it to work properly. Was the idea wrong?

Reply:
That wasn't clear to me. It seems two different things to me. In this case, ignore my comment^^.

Reply:
Um, I don't understand. It seems the issue in the fiddle can be fixed without enhancing the API of `PMREMGenerator`.

Reply:
Fiddle seems to offers more options than CodePen, so I'll look into that too. But I'm happy that it worked and that it can now be experienced live with the fiddle.
Thanks for doing that, Mugen

The one with the two issues: Of course thats already possible. If you just change the pmremNode without the sign correction in the x axis in the PMREMGenerator, then the x axis reflection is not correct and that is independent of the camera translation. 
The camera translation has no influence at all and is therefore actually a different topic. The bottom line for me is that it's just important to get it done properly and if those are two issues I can live with that very well.

I'm happy that the PMREMGenerator has come and the pmremNode, as the WebGLCubeRenderTarget previously seemed simply out of place in WebGPU, even though it worked well.